RELEASECHECKLIST.md

```diff
@@ -3,16 +3,24 @@
 ### Pre-Build
 * Update AssemblyInfo for kOS project
 * Update AssemblyInfo for kOS.Safe project
-* Update Resources\GameData\kOS\kOS.version
-* Update doc\source\conf.py
-* Update CHANGELOG.MD
+* Update AssemblyInfo for kOS.Safe.Test project
+* Update `Resources\GameData\kOS\kOS.version`
+* Update `doc\source\conf.py`
+* Update `CHANGELOG.MD`
+* Update `doc\source\changes.rst`
 
 ### Build
 * Build kOS solution in release mode
-* Ensure that all required resources are in place (ie KSP API Extensions DLL, ModuleManager DLL)
-* Create zip file with a root starting in the \Resources\ directory
-* Name the zip file with the following pattern kOS.v<major>.<minor>.<patch>.zip (eg kOS.v0.14.2.zip )
-* Build the documentation in \docs\
+* Run all unit tests
+* Ensure that all required resources are in place
+  * `Resources/GameData/kOS/Plugins/KSPAPIExtensions.dll`
+  * `Resources/GameData/ModuleManager.dll`
+* Create zip file with a root starting in the `\Resources\` directory
+* The zip file should have the GameData folder in the root
+* Name the zip file with the following pattern `kOS-v<major>.<minor>.<patch>.zip` (eg kOS-v0.14.2.zip )
+* Build the documentation in `\docs\`
```

Comment:
ah no, nevermind.  This is the 'how to release' checklist, not the instructions for newbs - so never mind, don't do that.